angular2-jwt.ts

```diff
@@ -18,21 +20,25 @@ export interface IAuthConfig {
  */
 
 export class AuthConfig {
-  
+
   config: any;
   headerName: string;
   headerPrefix: string;
   tokenName: string;
   tokenGetter: any;
+  refresh: any;
+  refreshOffset: number;
   noJwtError: boolean;
 
   constructor(config?: any) {
     this.config = config || {};
     this.headerName = this.config.headerName || 'Authorization';
-    this.headerPrefix = this.config.headerPrefix + ' ' || 'Bearer ';
+    this.headerPrefix = this.config.headerPrefix || 'Bearer';
     this.tokenName = this.config.tokenName || 'id_token';
-    this.noJwtError = this.config.noJwtError || false;
     this.tokenGetter = this.config.tokenGetter || (() => localStorage.getItem(this.tokenName));
+    this.refresh = this.config.refresh || (() => Observable.of(null));
```

Comment:
It was hard to think of a good default - I went with 'null' because if the token has expired and an attempt is made to refresh it but there is no refresh mechanism then the null token causes the same result as the original expired token.